rma/models/rma_order.py

```diff
@@ -175,8 +179,8 @@ def action_rma_done(self):
     def _get_valid_lines(self):
         """:return: A recordset of rma lines.
         """
-        self.ensure_one()
-        return self.rma_line_ids
+        for rec in self:
+            return rec.rma_line_ids
```

Comment:
Why this? I prefer the approach with the `ensure_one` if you call this method with a recordset a error must raise. If not, you can be disregarding errors.